include/soralog/sink.hpp

```diff
@@ -29,7 +29,7 @@ namespace soralog {
    */
   class Sink {
    public:
-    enum class ThreadFlag {
+    enum class UsingThreadInfo {
```

Comment:
It is more typical to use nouns for variable names. Personally I'm not quite getting what 'Using' means in this name.